Dockerfile

```diff
@@ -1,28 +1,28 @@
 FROM amazonlinux:1
 
 RUN \
-	yum install -y wget aws-cli python36 python36-devel git gcc &&	\
-	easy_install-3.6 pip
+  yum install -y wget aws-cli python36 python36-devel git gcc &&	\
+  easy_install-3.6 pip && \
+  yum clean all
 
 RUN	TOP=https://s3.amazonaws.com/cloudhsmv2-software/CloudHsmClient	\
-	VER=EL6								\
-	CLIENT=cloudhsm-client-3.1.0-3.el6.x86_64.rpm			\
-	PKCS11=cloudhsm-client-pkcs11-3.1.0-3.el6.x86_64.rpm;		\
-									\
-	for i in $CLIENT $PKCS11; do					\
-		wget "$TOP/$VER/$i";					\
-		yum install -y "$i";					\
-		rm -f "$i";						\
-	done
+  VER=EL6								\
+  CLIENT=cloudhsm-client-3.1.0-3.el6.x86_64.rpm			\
+  PKCS11=cloudhsm-client-pkcs11-3.1.0-3.el6.x86_64.rpm;		\
+  \
+  for i in $CLIENT $PKCS11; do					\
+  wget "$TOP/$VER/$i";					\
+  yum install -y "$i";					\
+  rm -f "$i";						\
+  done
```

Comment:
me editor autoformatted.... i can undo if desired

Reply:
hmmmm something really weird going on with the white spacing. i keep force pushing. sorry if someone is getting email spammed